opennms-web-api/src/main/java/org/opennms/web/svclayer/support/DefaultAdminApplicationService.java

```diff
@@ -203,9 +219,10 @@ public void performEditServices(String applicationIdString, String editAction, S
                 
                 service.addApplication(application);
                 m_monitoredServiceDao.save(service);
+                sentEvent(application, EventConstants.APPLICATION_CHANGED_EVENT_UEI);
```

Comment:
Same must be done in the performEditLocations() method, since the application is also saved there.

Reply:
fixed.